app/src/main/java/edu/grinnell/appdev/grinnelldirectory/APICaller.java

```diff
@@ -0,0 +1,138 @@
+package edu.grinnell.appdev.grinnelldirectory;
+
+import android.util.Log;
+
+import java.io.IOException;
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.APICallerInterface;
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.DatabaseAPI;
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+import edu.grinnell.appdev.grinnelldirectory.Model.User;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+
+
+public class APICaller {
+
+    private static final String BASE_URL = "https://itwebappstest.grinnell.edu/DotNet/WebServices/api/";
+
+    private Retrofit retrofit;
+    private DatabaseAPI dbAPI;
+    private Call<List<Person>> personQuery;
+    private APICallerInterface apiCallerInterface;
+    private User user;
+
+    public APICaller(User user1, APICallerInterface apiInterface) {
+        retrofit = new Retrofit.Builder().
+                baseUrl(BASE_URL).
+                addConverterFactory(GsonConverterFactory.create()).
+                build();
+        dbAPI = retrofit.create(DatabaseAPI.class);
+        apiCallerInterface = apiInterface;
+        user = user1;
+    }
+
+    /* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -  */
+    public void simpleSearch(User user, List<String> fields) {
+        personQuery = dbAPI.simpleSearch(user, fields.get(0), fields.get(1), fields.get(2), fields.get(3));
+
+        personQuery.enqueue(new Callback<List<Person>>() {
+
+            @Override
+            public void onResponse(Call<List<Person>> call, Response<List<Person>> response) {
+                if (response.isSuccessful()) {
+                    Log.d("API_SIMPLE_SUCCESS", "API returned list of people.");
```

Comment:
`Log`s should not be in production code. You should remove them when you're done verifying your responses